src/app/components/client/Chart.module.scss

```diff
@@ -10,6 +10,10 @@
     align-items: flex-end;
   }
 
+  button {
+    font: $text-body-xs;
```

Comment:
Remove, since the it seems like the button already has `size="small"` so it’s indeed not needed: [13889ee](https://github.com/mozilla/blurts-server/pull/3918/commits/13889eed6907886e4dbf5f91061d87df8b77c6d9).